### DISCUSSION TITLE
Lightswitch OS preference failing ESLint

### DISCUSSION THREAD

**Discussion by travisgalloway** [skeletonlabs/skeleton#2306] [Q&A]
The example code block for the OS preference for the lightswitch utility causes ESLint to fail parsing.

```
<svelte:head>{@html `<script>${autoModeWatcher.toString()} autoModeWatcher();</script>`}</svelte:head>
```

 I'm not sure how to correct or suppress the error. Any suggestions?

**Comment by endigo9740** [comment]
@travisgalloway hmm that's the first I've heard of this. I'll have to do a little more testing to determine what's up and provide a solution. Would you mind if I convert this to a bug ticket?

**Reply by travisgalloway** [reply]
Feel free. Thanks.
